build-system/pr-check.js

```diff
@@ -368,22 +374,12 @@ function main(argv) {
   if (process.env.BUILD_SHARD == "integration_tests") {
     // The integration_tests shard can be skipped for PRs.
     console.log(fileLogPrefix, 'Skipping integration_tests for PRs');
+    // Ideally, we'd run presubmit tests after `gulp dist`, as some checks run
```

Comment:
No, we are still running presubmit for PRs, just that it's run after `gulp build` since we're no longer doing `gulp dist`. This means some files that get generated by dist won't be checked during PRs, but will be checked on master.

This will change if and when we enable integration tests for PRs, at which point we will have to do a dist, and can run presubmit after that dist.